h2o-py/tests/testdir_algos/modelselection/pyunit_PUBDEV_8703_maxr_maxrSweep_compare.py

```diff
@@ -0,0 +1,35 @@
+import sys
+sys.path.insert(1, "../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.model_selection import H2OModelSelectionEstimator
+
+# compare mode maxr and maxrsweep.  They should generate the same models.
+def test_maxr_slow():
+    ncol = 189
+    train = h2o.create_frame("testFrame", 14200, ncol, factors=10, seed=1234)
```

Comment:
Wouldn't it be better to not set the seed to get bigger coverage by this test?

Reply:
@tomasfryda I am glad you bring up this issue.  I have argued with @michalkurka about this long time back.  His argument for wanting to use a seed is to make sure we have consistent test timing and results.  We agree to disagree on this issue.  Now I can add you on my side.